devtools/makefile_base_posix.mak

```diff
@@ -185,7 +185,7 @@ endif
 
 # If not specified by environment, use steam runtime compilers + in-tree ccache
 ifneq ($(filter default undefined,$(origin AR)),)
-	AR = $(STEAM_RUNTIME_PATH)/bin/ar crs
+	AR = $(STEAM_RUNTIME_PATH)/bin/ar rcs
```

Comment:
Need to figure out what the hell this `if` is checking for and make sure `ar` always has the commands added to it if needed